api/board.go

```diff
@@ -1,12 +1,14 @@
 package api
 
 import (
+	"strings"
+
 	"github.com/andygrunwald/go-jira"
 )
 
 func (client *Client) GetAllBoards(projectKeyOrID string, word string) ([]jira.Board, error) {
 	opts := jira.BoardListOptions{
-		BoardType:      "kanban",
+		BoardType:      "scrum",
```

Comment:
https://confluence.atlassian.com/jirasoftwarecloud/what-is-a-board-764477964.html
> Scrum board: For teams that plan their work in sprints. Includes a backlog.